docs/changes/newsfragments/5589.improved

```diff
@@ -0,0 +1,5 @@
+The use of multi index when exporting to xarray (and netcdf files) has been made smarter
+such that any dataset with a known shape (such as those measured by doNd etc) will
+never be exported using multi index even in the case of incomplete datasets (i.e. due to an interrupted measurement).
```

Comment:
if the use_multi_index is always, the export of a dataset with shape will still use multi index, while here it says "never" - so perhaps add "automatically" or rephrase in a different way, so that it's clear that the flag can be used to force whatever behavior the user desires?
```suggestion
never be automatically exported using multi index even in the case of incomplete datasets (i.e. due to an interrupted measurement).
```